lib/raidenclient/RaidenClient.ts

```diff
@@ -201,12 +203,35 @@ class RaidenClient extends SwapClient {
     // not implemented, raiden does not use invoices
   }
 
-  public getRoutes =  async (_amount: number, _destination: string) => {
-    // stub placeholder, query routes not currently implemented in raiden
-    // assume a fixed lock time of 100 Raiden's blocks
-    return [{
+  public getRoutes =  async (amount: number, destination: string, currency: string) => {
+    // a query routes call is not currently provided by raiden
+
+    /** A placeholder route value that assumes a fixed lock time of 100 Raiden's blocks. */
+    const placeholderRoute = {
       getTotalTimeLock: () => 101,
-    }];
+    };
+
+    if (this.directChannelChecks) {
+      // temporary check for a direct channel in raiden
+      const tokenAddress = this.tokenAddresses.get(currency);
+      const channels = await this.getChannels(tokenAddress);
+      for (const channel of channels) {
+        if (channel.partner_address && channel.partner_address === destination) {
+          const balance = channel.balance;
+          if (balance >= amount) {
```

Comment:
Not a major issue, but my first reaction here was that we're comparing  amount to units, but upon further inspection it turned out not to be the case.

We might want to consider calling these `unitAmounts` or `units`.

Reply:
Hmmm these are in satoshis though, whereas `units` elsewhere refer to the smallest base unit for the currency. If it's still confusing to you maybe I can clarify it another way, maybe just a line or two of comments explaining exactly what these lines do?

Reply:
Interesting... the `getRoutes` calls are made with units though:
https://github.com/ExchangeUnion/xud/blob/master/lib/swaps/Swaps.ts#L268
https://github.com/ExchangeUnion/xud/blob/master/lib/swaps/Swaps.ts#L529

Reply:
Think we need to convert the incoming units to satoshis then.

Reply:
Ahhh yes, good point, I will straighten this out.

Reply:
Per our discussion, I'll handle renaming these variables in a separate PR. These are both comparing base units as you originally stated.